### PR TITLE
round DPT 14 values

### LINKED ISSUE #583: Feature request: Rounding of sensor values according to KNX specification
XKNX sensors are sometimes displayed with lots of decimals, which does not make sense. Example: power = 21.123123123W

Sensors values should be rounded by XKNX according to the KNX standard. Example 0 decimals for DPT14.000 Unbenannt
For detailed specification see KNX System 3 Specifications -> 7 Interworking ->2 Datapoint Types

My proposal is to add an attribute resolution to each applicable dpt class (https://github.com/XKNX/xknx/blob/master/xknx/dpt).

```
class DPTAcceleration(DPT4ByteFloat):
    """DPT 14.000 DPT_Value_Acceleration (ms-2)."""

    dpt_main_number = 14
    dpt_sub_number = 0
    value_type = "acceleration"
    unit = "m/s²"
    resolution = 0
```

This attribute could be used afterwards in https://github.com/XKNX/xknx/blob/master/xknx/devices/sensor.py for rounding.

Commit https://github.com/XKNX/xknx/pull/304/commits/7687ef3f818950240e96a96979bdeb78b86cac23 would not be required.

Thanks for your comments

### PR DESCRIPTION
**Pull request by farmio**
## Description

Round DPT 14 values the way ETS does (presumably 🙃).

Test values are checked with ETS group monitor.

closes #583 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] The documentation has been adjusted accordingly
- [ ] The changes generate no new warnings
- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] The changes are documented in the changelog
- [ ] The Homeassistant plugin has been adjusted in case of new config options

### CODE REVIEW

File: test/devices_tests/sensor_expose_loop_test.py [hunk 1]
```diff
@@ -203,10 +203,10 @@ async def test_array_sensor_loop(self, value_type, test_payload, test_value):
         )
         await xknx.telegrams.join()
         xknx.knxip_interface.send_telegram.assert_called_with(outgoing_telegram)
-        if isinstance(test_value, float):
-            assert round(expose.resolve_state(), 4) == test_value
-        else:
-            assert expose.resolve_state() == test_value
+        # if isinstance(test_value, float):
+        #     assert round(expose.resolve_state(), 4) == test_value
+        # else:
```
Comment (marvin-w) [inline test/devices_tests/sensor_expose_loop_test.py:208]: consider removing commented out code, git history has it all

File: test/devices_tests/sensor_expose_loop_test.py [hunk 2]
```diff
@@ -237,10 +237,10 @@ async def test_array_sensor_loop(self, value_type, test_payload, test_value):
             ]
         )
         # test if Sensor has successfully read from ExposeSensor
-        if isinstance(test_value, float):
-            assert round(sensor.resolve_state(), 4) == test_value
-        else:
-            assert sensor.resolve_state() == test_value
+        # if isinstance(test_value, float):
+        #     assert round(sensor.resolve_state(), 4) == test_value
+        # else:
```
Comment (marvin-w) [inline test/devices_tests/sensor_expose_loop_test.py:242]: consider removing commented out code

File: changelog.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased changes
 
+### Internals
+
+- Round DPT 14 values to precicion of 7 digits
```
Comment (marvin-w) [inline changelog.md:7]:
```suggestion
- Round DPT 14 values to precision of 7 digits
```

**Review by marvin-w — APPROVED** [review]
Looks good to me, just some minor typos

File: test/devices_tests/sensor_expose_loop_test.py (hunk 1 above)
Comment (farmio) [reply test/devices_tests/sensor_expose_loop_test.py:208]: oops...

**Pull request merged**